tests/integration/questionnaire/test_questionnaire_placeholders.py

```diff
@@ -42,12 +42,6 @@ def test_placeholders_rendered_in_pages(self):
 
         self.post({"total-retail-turnover-internet-sales-answer": 3000})
 
-        self.post({"additional-family-members-answer": 12})
-
-        self.assertInBody("Your family size is <em>13</em>")
-
-        self.post()
-
         self.post({"total-items-answer": 2})
 
         self.assertInBody("Do you want to add <em>a 3rd</em> item?")
```

Comment:
This test should also test that the placeholder '3rd' is rendered on the summary.

Reply:
I've added it, thanks👍